master/getting-started/bare-metal/bare-metal.md

```diff
@@ -387,12 +387,23 @@ To avoid completely cutting off a host via incorrect or malformed
 policy, {{site.prodname}} has a failsafe mechanism that keeps various pinholes open
 in the firewall.
 
-By default, {{site.prodname}} keeps port 22 inbound open on *all* host endpoints,
-which allows access to ssh; as well as inbound and outbound communications to
-ports 2379, 2380 (etcd's default ports), and to ports 6666, 6667 (etcd's ports
-as deployed by {{site.prodname}}'s self-hosted Kubernetes manifests).
+By default, {{site.prodname}} keeps the following ports open:
```

Comment:
Last nit, I think. The previous text emphasizes "all host endpoints". We should probably retain that piece of information as-is.

Reply:
that's a good point! I've added that line back, PTAL